src/printer/smt2/smt2_printer.cpp

```diff
@@ -243,6 +243,12 @@ void Smt2Printer::toStream(std::ostream& out,
       out << smtKindString(n.getConst<Kind>(), d_variant);
       break;
     case kind::CONST_RATIONAL: {
+      const Rational& r = n.getConst<Rational>();
+      toStreamRational(out, r, true, d_variant);
```

Comment:
I'd suggest to rename the `decimal` argument to something else. It's more a `forceReal` or alike